backend.rb

```diff
@@ -57,14 +56,13 @@ class Backend < Sinatra::Base
       origin_proof:  params[:origin_proof]
     )
 
-    authorization_response = purchase.authorize!
-    if authorization_response.success?
+    success, response_data, response_code = purchase.order!
+    if success
       @user.subscribe!
       article = Article.find(params[:article_id])
       return json data: article.paid_content
```

Comment:
@eluxen also in redirect_to.rb page?

return json redirect_to: "/thank_you?user_token=#{user_token}&order_reference=#{order_reference}", klarna_response: response_data ?